test/SlowTests/Issues/RavenDB-15330.cs

```diff
@@ -33,69 +20,26 @@ public RavenDB_15330(ITestOutputHelper output) : base(output)
     [Fact]
     public async Task Command_With_DontCareId_Should_Be_Committed()
     {
-        var typeProp = nameof(RachisLogHistory.LogHistoryColumn.Type);
-        var stateProp = nameof(RachisLogHistory.LogHistoryColumn.State);
-        var indexProp = nameof(RachisLogHistory.LogHistoryColumn.Index);
-
         var server = GetNewServer();
-        using var store = GetDocumentStoreForRollingIndexes(new RavenTestBase.Options
+        using var store = GetDocumentStoreForRollingIndexes(new Options
         {
             ModifyDatabaseRecord =
                 record => record.Settings[RavenConfiguration.GetKey(x => x.Indexing.MaxTimeToWaitAfterFlushAndSyncWhenReplacingSideBySideIndex)] = "5",
             Server = server,
             ReplicationFactor = 1,
         });
 
-        await store.ExecuteIndexAsync(new SomeRollingIndex());
-
-        var result = await server.ServerStore.SendToLeaderAsync(new PutRollingIndexCommand(store.Database, nameof(SomeRollingIndex),
+        await server.ServerStore.SendToLeaderAsync(new PutRollingIndexCommand(store.Database, "SomeRollingIndex",
             DateTime.UtcNow, RaftIdGenerator.DontCareId));
 
         using (server.ServerStore.Engine.ContextPool.AllocateOperationContext(out ClusterOperationContext context))
         using (context.OpenReadTransaction())
         {
             var historyLog = server.ServerStore.Engine.LogHistory.GetHistoryLogs(context);
-            bool contain = false;
-            bool first = true;
-            long lastIndex = 0;
-            foreach (var djv in historyLog)
-            {
-                if (first)
-                {
-                    lastIndex = long.Parse(djv[indexProp].ToString());
-                    first = false;
-                }
-                else
-                {
-                    var currentIndex = long.Parse(djv[indexProp].ToString());
-                    Assert.Equal(lastIndex+1, currentIndex);
-                    lastIndex = currentIndex;
-                }
-
-                if (djv[typeProp] != null && djv[typeProp].ToString() == nameof(PutRollingIndexCommand) && 
-                    djv[stateProp] !=null && djv[stateProp].ToString() == "Committed")
-                {
-                    contain = true;
-                    break;
-                }
-            }
+            bool contain = historyLog.Any(djv => djv[nameof(RachisLogHistory.LogHistoryColumn.Type)]?.ToString() == nameof(PutRollingIndexCommand) && 
+                                                 djv[nameof(RachisLogHistory.LogHistoryColumn.State)]?.ToString() == "Committed");
```

Comment:
I think instead of all this we can use `WaitForRaftCommandToBeAppliedInClusterAsync` or `WaitForRaftCommandToBeAppliedInLocalServerAsync`?

Reply:
done